common/attest_plugin.c

```diff
@@ -312,6 +312,15 @@ oe_result_t oe_verify_evidence(
                 evidence->version,
                 OE_ATTESTATION_HEADER_VERSION);
 
+        if (evidence_buffer_size !=
+            (evidence->data_size + sizeof(oe_attestation_header_t)))
+            OE_RAISE_MSG(
+                OE_INCORRECT_REPORT_SIZE,
```

Comment:
```suggestion
                OE_INVALID_PARAMETER,
```
Is `OE_INCORRECT_REPORT_SIZE` intended to be a legacy error code for the legacy `oe_get_report()`?